src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header">
+        <div class="headerContent">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </div>
+        <nav>
+          <ul class="headerContent">
+            <li>
+              <a
+                href="a"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'nav_links' should be renamed to something more meaningful.